_assets/ci/Jenkinsfile.linux-nix

```
@@ -0,0 +1,37 @@
+#!/usr/bin/env groovy
+library 'status-jenkins-lib@v1.9.13'
+
+pipeline {
+  agent { label 'linux && x86_64 && nix-2.24' }
+
+  options {
+    disableConcurrentBuilds()
+    /* manage how many builds we keep */
+    buildDiscarder(logRotator(
+      numToKeepStr: '20',
+      daysToKeepStr: '30',
+    ))
+  }
+
+  stages {
+    stage('Build status-go library') {
+      steps {
+        script {
+          nix.flake("status-go-library")
+        }
+      }
+    }
+
+    stage('Build status-go mobile Android library') {
```

We know we are building `status-go`, no need to repeat it in every stage name.
```suggestion
    stage('Build mobile Android library') {
```